ZipDir.go

```diff
@@ -84,6 +84,8 @@ func (this *ZipDir) ReadArchive() error {
 		attrs := Attrs{
 			Mode:   zipFile.Mode(),
 			Mtime:  zipFile.ModTime(),
```

Comment:
what about subdir?

Reply:
Added tests for subdir and files in subdir.